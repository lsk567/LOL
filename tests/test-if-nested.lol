bool flag1 = false;
bool flag2 = true;
if (flag1){
    	if (flag2) {
		println("both true");
	} else {
		println("flag1 true, flag2 false");
	}
}
else{
	if (flag2) {
		println("flag1 false, flag2 true");
	} else {
		println("both false");
	}
}

