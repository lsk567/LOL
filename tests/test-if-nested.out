flag1 false, flag2 true
